ios/RNDateTimePickerShadowView.m

```diff
@@ -0,0 +1,52 @@
+#import "RNDateTimePickerShadowView.h"
+
+@implementation RNDateTimePickerShadowView
+
+- (instancetype)init
+{
+  if (self = [super init]) {
+    YGNodeSetMeasureFunc(self.yogaNode, RNDateTimePickerShadowViewMeasure);
+  }
+  return self;
+}
+
+- (void)setDate:(NSDate *)date {
+  _date = date;
+  YGNodeMarkDirty(self.yogaNode);
```

Comment:
Since different date can change the layout of the picker, we need to mark the Yoga node dirty.